benchmark_runner/common/template_operations/templates/vdbench/internal_data/vdbench_pod_template.yaml

```diff
@@ -1,13 +1,19 @@
+{% if not scale %}
```

Comment:
Why aren't you using a namespace for the scaling test?

Reply:
We're running everything else in the benchmark-runner namespace; why not this?

Reply:
That's OK, but shouldn't they all be in the benchmark-runner namespace?

Reply:
Yes, all in benchmark-runner namespace. 
Need to create the namespace once for scale, so I have added it to redis yaml.

Reply:
In redis??  I think it makes more sense to always create the namespace here, rather than having more conditionalized code.

Reply:
Yes in scale we create it in redis because we created redis first.

Reply:
OK